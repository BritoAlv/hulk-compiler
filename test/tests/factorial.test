function factorial(n : number) : number => 
if (n < 1)
    1
else
    n * factorial(n - 1);

print(factorial(5));