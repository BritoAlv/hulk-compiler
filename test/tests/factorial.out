120.00000000

